data/fields/healthcare/speciality_dentist.json

```
@@ -0,0 +1,12 @@
+{
+    "key": "healthcare:speciality",
+    "type": "semiCombo",
+    "label": "Specialties",
```

```suggestion
    "label": "Specialties",
    "autoComplete": false
```
This should make it that only the values defined below (plus any that are already present) are supported. But I am on my phone so there might be a typo. Best to check the /schema repo docs.